### PR TITLE
NGINX-Ingress-controller - Variables: QuickStart Enhancement

### PR DESCRIPTION
**Pull request by gsidhwani-nr**
…d added small sized logo

Updated-  Config.yml, Dashboard for variables , dashboard snapshot and added small sized logo

# Summary

A concise description of the changes being introduced. Please review the pre-merge checklist section to validate this pull request is ready for review and merge. **If it is not ready, please mark the pull request as a draft.**

## Pre merge checklist

- [ X] Did you check you NRQL syntax? - Does it work?
- [ X] Did you include an InstallPlan and Documentation reference?
- [X ] Are all documentation links publically accessible?
- [ X] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)? 
- [ X] Did you check your descriptive content for spelling and grammar errors?

### Dashboards 

- [X ] Does the PR contain a screenshot for each of your dashboards?
- [X ] Do your screenshots show data?
- [X ] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

https://staging.onenr.io/0a7j911OljO

### Alerts

- [ X] Did you check that your alerts actually work?

### CODE REVIEW

**Review by thezackm — APPROVED** [review]
approved for the `prometheus-growth` project

File: quickstarts/kubernetes/nginx/config.yml [hunk 1]
```diff
@@ -36,16 +37,20 @@ keywords:
   - config
 
 installPlans:
-  - third-party-nginx-ingress
+  - setup-prometheus
+  - setup-prometheus-agent-mode
```
Comment (aswanson-nr) [inline quickstarts/kubernetes/nginx/config.yml:41]: `setup-prometheus-agent-mode` does not appear to be an install plan. Could this be a typo?

**Review by aswanson-nr — CHANGES_REQUESTED** [review]
This looks pretty much ready! Just found one issue with the install plan

File: dashboards/nginx-ingress/nginx-ingress.json [hunk 2]
```diff
@@ -1,443 +1,602 @@
...
+        {
+          "value": {
+            "string": "*"
+          }
+        }
+      ],
+      "nrqlQuery": {
+        "accountIds": [
+          0
+        ],
```
Comment (thezackm) [inline dashboards/nginx-ingress/nginx-ingress.json:522]: set to `"accountIds": []`

File: dashboards/nginx-ingress/nginx-ingress.json [hunk 3]
```diff
@@ -1,443 +1,602 @@
...
```
Comment (thezackm) [inline dashboards/nginx-ingress/nginx-ingress.json:543]: set to `"accountIds": []`

File: dashboards/nginx-ingress/nginx-ingress.json [hunk 4]
```diff
@@ -1,443 +1,602 @@
...
```
Comment (thezackm) [inline dashboards/nginx-ingress/nginx-ingress.json:564]: set to `"accountIds": []`

File: quickstarts/kubernetes/nginx/config.yml (hunk 1 above)
Comment (thezackm) [reply quickstarts/kubernetes/nginx/config.yml:43]: pending in https://github.com/newrelic/newrelic-quickstarts/pull/1503

**Review by thezackm — CHANGES_REQUESTED** [review]
@gsidhwani-nr - changes requested; ty

File: dashboards/nginx-ingress/nginx-ingress.json (hunk 2 above)
Comment (gsidhwani-nr) [reply dashboards/nginx-ingress/nginx-ingress.json]: Modified.

File: dashboards/nginx-ingress/nginx-ingress.json (hunk 3 above)
Comment (gsidhwani-nr) [reply dashboards/nginx-ingress/nginx-ingress.json]: Modified.

**Review by thezackm — APPROVED** [review]

**Review by josephgregoryii — APPROVED** [review]

**Pull request merged**